src/pykiso/test_coordinator/test_result.py

```diff
@@ -138,6 +143,9 @@ def addFailure(self, test: TestCase, err: tuple) -> None:
         TestResult.addFailure(self, test, err)
         self._error_occurred = True
 
+    def addSuccess(self, test: TestCase):
+        self.successes.append(test)
```

Comment:
Even if this is quite obvious it still needs a short docstring

Reply:
ok